kde/kdenetwork/kio-extras/kio-extras.py

```diff
@@ -12,6 +12,7 @@ def setTargets(self):
     def setDependencies(self):
         self.buildDependencies["virtual/base"] = None
         self.buildDependencies["kde/frameworks/extra-cmake-modules"] = None
+        self.buildDependencies["lib/libssh"] = None
```

Comment:
Hi thanks for your contribution.
As we will also need the dependency during runtime it is a runtimeDependency.